examples/pytorch/gat/main.py

```diff
@@ -22,23 +35,25 @@ def init_args(parser):
     parser.add_argument("--head_num", type=str2list_int, default="8,1", help="the number of attention headers.")
     parser.add_argument("--hidden_dim", type=int, default=8, help="hidden layer dimension.")
     parser.add_argument("--num_classes", type=int, default=-1, help="number of classes for category")
-    parser.add_argument("--ffd_drop", type=float, default=0.0, help="feature dropout rate.")
-    parser.add_argument("--attn_drop", type=float, default=0.0, help="attention layer dropout rate.")
+    parser.add_argument("--ffd_drop", type=float, default=0.6, help="feature dropout rate.")
+    parser.add_argument("--attn_drop", type=float, default=0.6, help="attention layer dropout rate.")
     parser.add_argument("--l2_coef", type=float, default=0.0005, help="l2 loss")
 
     # GAT Query part
     parser.add_argument("--neighbor_edge_types", type=str2list_int, default="0", help="Graph Edge for attention encoder.",)
-
-    # evaluate node file.
-    parser.add_argument("--eval_file", default="", type=str, help="")
 # fmt: on
 
 
-def create_model(args: argparse.Namespace):
-    get_logger().info(f"Creating GAT model with seed:{args.seed}.")
-    # set seed before instantiating the model
+def train_func(config: Dict):
```

Comment:
this looks exactly like train_func in `ray_train.py`. Why do we need to copy it instead of reusing?